### ISSUE TITLE
invalid video 

### ISSUE THREAD

**Issue by abdulazizalmass** [HarshMarolia/Youtube-Playlist-Downloader#1]
Amazing CSS. Wow.

Only issue is that I tried with multiple playlist with no luck. error `invalid video`

**Comment by ZeroNoKotae** [comment]
Yep invalid video. maybe something is messed up with youtube api?
